fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MaterializedViewRewriter.java

```diff
@@ -149,35 +171,31 @@ public List<OptExpression> rewrite(ReplaceColumnRefRewriter queryColumnRefRewrit
             return Lists.newArrayList();
         }
         ScalarOperator mvPredicate = MvUtils.rewriteOptExprCompoundPredicate(mvExpression, mvColumnRefRewriter);
+
         if (!ConstantOperator.TRUE.equals(mvPartitionPredicate)) {
+            // add latest partition predicate to mv predicate
```

Comment:
It looks like not suitable for comment here

Reply:
fixed